support/sier-codec/src/lib.rs

```diff
@@ -48,14 +50,76 @@ impl Parser {
     pub fn struct_def(&self, name: &str) -> Option<&Arc<StructDef>> {
         self.structs.values().find(|s| s.type_name() == name)
     }
-}
 
-impl Default for Parser {
-    fn default() -> Self {
-        Parser {
-            structs: HashMap::new(),
+    fn transform_serde_value<'a>(
+        &'a self,
+        value: &SerdeValue,
+        type_: &'a Type,
+    ) -> Result<Value<'a>, Error<'a>> {
+        match value {
+            SerdeValue::Null => Ok(Value::Unit),
+            //TODO(melatron): Implement a boolean type for sier
+            SerdeValue::Bool(_bool) => Ok(Value::Unit),
```

Comment:
Keep the arms of this match as `unimplemented!()` for now where we don't have a good implementation. Null, Bool, Number<float>.